src/Services/LightningService.cs

```diff
@@ -479,21 +401,6 @@ await foreach (var response in openStatusUpdateStream.ResponseStream.ReadAllAsyn
                                         }
                                     }
 
-                                    //We check that the partial signatures number has changed, otherwise finalize inmediately
-                                    var partialSigsCountAfterSignature =
-                                        finalSignedPSBT.Inputs.Sum(x => x.PartialSigs.Count);
-
-                                    if (partialSigsCountAfterSignature == 0 ||
-                                        partialSigsCountAfterSignature <= partialSigsCount)
-                                    {
-                                        var invalidNoOfPartialSignatures =
-                                            $"Invalid expected number of partial signatures after signing the PSBT";
-
-                                        _logger.LogError(invalidNoOfPartialSignatures);
-                                        throw new ArgumentException(
-                                            invalidNoOfPartialSignatures);
-                                    }
-                                    
```

Comment:
Why is this block removed?

Reply:
Because is repeated inside the SignPSBT function